seq2seq/evaluator/evaluator.py

```diff
@@ -48,4 +48,6 @@ def evaluate(self, model, data):
                 # Evaluate loss
                 loss.eval_batch(batch_output, batch_target)
 
+        model.train(True)
```

Comment:
@kylegao91 since evaluator should never evaluate with dropout, I would recommend not setting `model.train(True)` in here. Let the trainer handle it. 

I prefer only setting `model.eval()` in the evaluator.